From: Benjamin Kaduk <kaduk@mit.edu>
Date: Mon, 8 Sep 2014 13:42:27 -0400
Subject: Let mancheck_utils ignore version subcommands

We don't have a man page for the 'version' subcommand, which has
"always" been present but only recently was exposed to the usage.
It's okay to not have a man page for it, so tell the test infrastructure
to not complain about its absence.

Change-Id: Ife834d41797d1d1efe403b204736ac85d62724e9
(cherry picked from commit 311626477af18dff1aa9ecb67545e22985b1824e)
---
 tests/tests-lib/perl5/mancheck_utils.pm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/tests-lib/perl5/mancheck_utils.pm b/tests/tests-lib/perl5/mancheck_utils.pm
index 6b56274..edfa8fc 100644
--- a/tests/tests-lib/perl5/mancheck_utils.pm
+++ b/tests/tests-lib/perl5/mancheck_utils.pm
@@ -35,6 +35,8 @@ sub lookup_sub_commands {
     while (<HELPOUT>) {
         # Skip the header thingy
         next if /Commands are/;
+        # Skip the version subcommand, it's always present but not interesting
+        next if /^version/;
         @comm = split();
         push(@subcommlist, $comm[0]);
     }
